tests/integration/nm/wired_test.py

```diff
@@ -0,0 +1,68 @@
+#
+# Copyright 2019 Red Hat, Inc.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+
+from libnmstate import nm
+from libnmstate import schema
+
+from .testlib import mainloop
+
+
+ETH1 = 'eth1'
+
+MTU0 = 1200
+
+
+def test_interface_mtu_change(eth1_up):
```

Comment:
Will this test restore the previous MTU once the test is done?

Reply:
The eth1_up fixture hopefully does it by deleting the profile and redefining it.

Reply:
Confirmed: Any test which uses `eth1` will have to call `eth1_up` (or `eth1_down`) and that will reset the mtu value (like all the other values it may have).